main.cc

```diff
@@ -244,6 +244,18 @@ verify_rlimit(bool developer_mode) {
     if (r == -1) {
         throw std::system_error(errno, std::system_category());
     }
+
```

Comment:
nitpick: I can't place a comment in this function's header (thx github), so I'll do it here - perhaps the name should no longer be `verify_rlimit`, it sounds read-only, and this function actually changes the configuration

Reply:
I'll change to adjust_and_verify_rlimit().